Client.xcodeproj/project.pbxproj

```diff
@@ -241,6 +241,7 @@
 		3D9CA9A81EF84D04002434DD /* NoImageTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 3D9CA9A71EF84D04002434DD /* NoImageTests.swift */; };
 		3D9CAA1C1EFCD655002434DD /* ClipBoardTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 3D9CAA1B1EFCD655002434DD /* ClipBoardTests.swift */; };
 		3DEFED081F55EBE300F8620C /* TrackingProtectionTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 3DEFED071F55EBE300F8620C /* TrackingProtectionTests.swift */; };
+		3E481B3624938A1A00DA673C /* Metrics.swift in Sources */ = {isa = PBXBuildFile; fileRef = 3E481B3524938A1A00DA673C /* Metrics.swift */; };
```

Comment:
Unrelated to the patch, but I had to do this to make FxiOS compile. I'm not sure how it worked before, but it looks like `LegacyIds` now gets declared in `Metrics.swift` instead of its own file?

Reply:
LegacyIds is in Metrics.swift. This change is adding it as a build file to a different target, it should be removed from its location in the Client target. Here is how it looks on master, in which it is in the wrong place, it should be part of Shared target, which I assume is the change here, but I can't tell just from reading the diff.
<img width="264" alt="image" src="https://user-images.githubusercontent.com/5495083/85332307-b7887b80-b4a5-11ea-86bb-8f4a0542e632.png">

Reply:
Hmm, adding it to `Shared` makes a recursive dependency. I just left it alone.